docs/usage-with-routing.stories.mdx

```diff
@@ -38,11 +38,19 @@ const CarbonButtonWithReactRouter = ({ to, children }) => {
 
 const CarbonLinkWithReactRouter = ({ to, children }) => {
   const history = useHistory();
-  const onClick = useCallback(() => {
-    history.push(to);
-  }, [to, history]);
+  const onClick = useCallback(
+    (event) => {
+      event.preventDefault();
+      history.push(to);
+    },
+    [to, history]
+  );
 
-  return <Link onClick={onClick}>{children}</Link>;
+  return (
+    <Link onClick={onClick} href={to}>
```

Comment:
I'm not sure this is right - now both the href and the `onClick` will be navigating, it should be one or the other I think